airdrop_utils/scripts/run.js

```diff
@@ -0,0 +1,44 @@
+const { ethers } = require("hardhat");
+const hre = require("hardhat");
+
+async function main() {
+
+  // Define a list of wallets to airdrop NFTs
+  // Note: These should be 32Byte Spanning Addresses
+  // https://docs.spanning.network/docs/concepts/address
+  const airdropAddresses = [
```

Comment:
since we're in javascript, can we use our Spanning utils to create these? they'll still have a lot of constants hard-coded, but hopefully it'll make it more clear?

Reply:
Done